core/modules/utils/utils.js

```diff
@@ -799,5 +799,22 @@ exports.strEndsWith = function(str,ending,position) {
 		return lastIndex !== -1 && lastIndex === position;
 	}
 };
+	
+/**
+ * @function setTags
+ * @description Helper function to override the tags of a tiddler
+ * If the tiddler does not exist it will be created
```

Comment:
If tiddlers are created, they need the `created` and `modified` fields, otherwise, they don't show up in the "recent" tab
If tiddlers are changed, the modified field has to be updated

Reply:
What do you think will be best ? To just handle the update and if the toddler does not exist return false (true in case of update) or to handle both scenarios ?